lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -20,7 +20,7 @@ function Get-PlatformVersion {
 }
 
 function Get-PlatformArchitecture {
-  if ((Get-Win32OS).osarchitecture -like '64-bit') {
+  if ((Get-Win32OS).osarchitecture -match '64') {
```

Comment:
ARM64 is not a concern, right?

Reply:
We would need further modifications to this script to support arm64 in any case, and this change will not make that situation any worse.